hack/BUILD.bazel

```diff
@@ -26,6 +26,16 @@ sh_test(
     tags = ["lint"],
 )
 
+sh_test(
+    name = "verify-shellcheck",
+    srcs = ["verify-shellcheck.sh"],
```

Comment:
Can we get this to work on a mac too?

Reply:
I can't find any osx binaries, unfortunately. We could make the test only run on Linux, maybe?

Reply:
yeah, shellcheck is commonly installed via brew on darwin: https://formulae.brew.sh/formula/shellcheck. I don't know that upstream distributes binaries for it.